webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala

```diff
@@ -92,6 +92,8 @@ class UsersResponderADM extends Responder {
 
         //log.debug("usersGetV1")
 
+        // ToDO: need to have certain permissions to allow access
```

Comment:
Is there an issue for this?

Reply:
yes, should probably add the link?

Reply:
#961

Reply:
I like to add the issue number in the TODO, but I think the important thing is just that there's an issue, because it's easy to forget TODOs.

Reply:
issue number added.